scripts/monorepo/checkIfPackagesAffected.js

```diff
@@ -0,0 +1,20 @@
+const getAffectedPackages = require('./getAffectedPackages');
+
+const FLUENTUI_REACT = '@fluentui/react';
+const FLUENTUI_REACT_COMPONENTS = '@fluentui/react-components';
+const FLUENTUI_REACT_NORTHSTAR = '@fluentui/react-northstar';
+const packages = [FLUENTUI_REACT, FLUENTUI_REACT_COMPONENTS, FLUENTUI_REACT_NORTHSTAR];
+
+const isPackageAffected = packagesToCheck => {
+  for (const pkg of packagesToCheck) {
+    if (getAffectedPackages().has(pkg)) {
+      if (pkg === FLUENTUI_REACT || pkg === FLUENTUI_REACT_COMPONENTS) {
```

Comment:
suggestion: this contains lot of if , if else branching that is not necessary. the whole implementation could be be done without deep branching, via functional approach:

```js
const isPackageAffected = packagesToCheck => {
  const affected = getAffectedPackages();

  packagesToCheck.forEach(pkgName => {
    if (!affected.has(pkg)) {
      // no operation needed
      return;
    }

    if (pkg === FLUENTUI_REACT || pkg === FLUENTUI_REACT_COMPONENTS) {
      // do stuff
      return;
    }
    if (pkg === FLUENTUI_REACT_NORTHSTAR) {
      // do stuff
      return;
    }
  });
};
```

or via imperative approach ( less preferred ) 

```js
const isPackageAffected2 = packagesToCheck => {
  const affected = getAffectedPackages();

  for (const pkg of packagesToCheck) {
    if (!affected.has(pkg)) {
      continue;
    }

    if (pkg === FLUENTUI_REACT || pkg === FLUENTUI_REACT_COMPONENTS) {
      // do work
      continue;
    }

    if (pkg === FLUENTUI_REACT_NORTHSTAR) {
      // do work
      continue;
    }
  }
};
```

Reply:
Went with imperative approach since in this case, I don't want to return immediately. Need to check if the other packages were also affected and set its corresponding ADO variable appropriately.